clang-tools-extra/clangd/unittests/PrerequisiteModulesTest.cpp

```diff
@@ -402,6 +402,46 @@ import A;
   EXPECT_TRUE(D.isFromASTFile());
 }
 
+// An end to end test for code complete in modules
+TEST_F(PrerequisiteModulesTests, CodeCompleteTest) {
```

Comment:
can you also add a test for signature help?

Reply:
Done